target/linux/octeon/files/arch/mips/boot/dts/cavium-octeon/n821.dts

```diff
@@ -0,0 +1,363 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+/dts-v1/;
+
+/ {
+        model = "Cisco/Viptela vEdge 1000";
+        compatible = "n821";
```

Comment:
That's not a proper compatible. It should have the form of `"<vendor>,<model>"`, and you also should have an appropriate compatible string for the soc (e.g. "cavium,cn7130"). Take a look at the other dts files in this directory.

Similarily, the dts file name should match the compatible naming as <soc>_<compatible>.dts.